src/op_mode/nhrp.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2023 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+import tabulate
+import vyos.opmode
+
+from vyos.util import cmd
+from vyos.util import process_named_running
+
+
+def get_formatted_output(output_list: list) -> str:
+    """
+    Create formatted table for CLI output
+    :param output_list: list for API
+    :type output_list: list
+    :return: tabulate string
+    :rtype: str
+    """
+    status: dict = output_list.pop(0)
+    if 'Status' in status:
+        print(f"Status: {status['Status']}")
+    output: str = tabulate.tabulate(output_list, headers='keys',
+                                    numalign="left")
+    return output
+
+
+def get_formatted_list(output_string: str) -> list:
+    """
+    Format string returned from CMD to API list
+    :param output_string: String received by CMD
+    :type output_string: str
+    :return: list for API
+    :rtype: list
+    """
+    formatted_list: list = []
+    output_list: list = output_string.split('\n\n')
+    for list_a in output_list:
```

Comment:
We have a function for converting colon-separated lists to dicts: [vyos.util.colon_separated_to_dict](https://github.com/vyos/vyos-1x/blob/current/python/vyos/util.py#L320).

Reply:
done